user-console/src/main/java/org/pentaho/mantle/client/commands/NewFolderCommand.java

```diff
@@ -100,7 +100,7 @@ protected void performOperation( boolean feedback ) {
     event.setAction( this.getClass().getName() );
 
     final TextBox folderNameTextBox = new TextBox();
-    folderNameTextBox.setTabIndex( 1 );
+    folderNameTextBox.setTabIndex( 0 );
```

Comment:
AFAIK, tab index 0 is the default for text boxes. So you should just remove the statement.